### DISCUSSION TITLE
Sorting book in series

### DISCUSSION THREAD

**Discussion by kayhos** [gotson/komga#1635] [Q&A] [answered]
I have a large manga library and some books have only a chapter assigned to them (no volume has been produced yet) and some other have a volume and a chapter. When Komga presents the books in a library setup as above, all the volumeless chapters get presented before chapters with volume.

For example, I end up having Chapter 432 at the start of the series and Volume 1 Chapter 1 in the lasts books.

Would it be possible to sort the books in a manga serie based on chapter instead of alphabetically ?

**Comment by gotson** [comment, marked as answer]
This is a naming issue, use the sort number field in the book metadata to sort the chapters after the volumes, or rename your files.
